warp/payload/payload.go

```diff
@@ -44,7 +44,7 @@ func ParseAddressedPayload(b []byte) (*AddressedPayload, error) {
 }
 
 // Initialize recalculates the result of Bytes().
```

Comment:
nit: should update the casing of the comment

Reply:
Good call